comic/api/mutations.py

```diff
@@ -0,0 +1,96 @@
+import graphene
+
+from comic.api.types import ComicBookInput, ComicBookType, DirectoryInput, DirectoryType
+from comic.models import ComicBook, Directory
+
+
+class CreateDirectory(graphene.Mutation):
+    class Arguments:
+        input = DirectoryInput(required=True)
+
+    ok = graphene.Boolean()
+    directory = graphene.Field(DirectoryType)
+
+    @staticmethod
+    def mutate(root, info, input=None):
+        ok = True
+        directory = Directory(name=input.name)
```

Comment:
There is no reference to a parent directory here. You will only be able to make dirs at the root level